api/spec/v1.yaml

```
@@ -2051,6 +2051,12 @@ components:
           type: integer
           format: uint64
           description: The total gas used by the transaction.
+        charged_fee:
+          type: string
+          description: |
+            The fee that was charged for the transaction execution (total, native denomination,
+            ParaTime base units, as a string).
+            Calculated from gas price (=fee/gas_limit) and gas used.
```

```suggestion
            Calculated as `gas_price * gas_used`, where `gas_price = fee / gas_limit`.
```

Let's be explicit since there's so much related terminology here.